docs/upgrade.html

```diff
@@ -73,7 +73,11 @@ <h5><a id="upgrade_320_notable" href="#upgrade_320_notable">Notable changes in 3
             via Connect worker and/or connector configuration. Connect may enable idempotent producers
             by default in a future major release.</li>
         <li>Kafka has replaced log4j and slf4j-log4j12 with reload4j and slf4j-reload4j due to security concerns.
```

Comment:
Maybe we can say something like:

>         <li>Kafka has replaced log4j and slf4j-log4j12 with reload4j and slf4j-reload4j due to security concerns.
>              This only affects modules that specify a logging backend (`connect-runtime` and `kafka-tools` are two such
>              examples). A number of modules, including `kafka-clients`, leave it to the application to specify the logging
>              backend. More information can be found at <a href"https://reload4j.qos.ch">reload4j</a>.
>             Projects that depend on the affected modules from the Kafka project should use
>             <a href="https://www.slf4j.org/manual.html#swapping">slf4j-log4j12 version 1.7.35 or above</a> or
>             slf4j-reload4j to avoid
>             <a href="https://www.slf4j.org/codes.html#no_tlm">possible compatibility issues originating from the logging framework</a>.

Reply:
We should also update the same text in the 3.1 branch once we merge this.

Reply:
@ijuma changed the text as you proposed.

I will cherry-pick the commit to 3.2 and 3.1.

@tombentley Since this is just a doc change, we do not need a new RC, right? We can port the change directly to the doc repo in case RC1 passes the votes.